collect_app/src/androidTest/java/org/odk/collect/android/support/pages/ProjectSettingsDialogPage.kt

```diff
@@ -45,8 +45,8 @@ internal class ProjectSettingsDialogPage() : Page<ProjectSettingsDialogPage>() {
         return this
     }
 
-    fun assertNotInactiveProject(projectName: String, subtext: String): ProjectSettingsDialogPage {
-        onView(allOf(hasDescendant(withText(projectName)), hasDescendant(withText(subtext)), withContentDescription(getTranslatedString(R.string.switch_to_project, projectName)))).check(doesNotExist())
+    fun assertNotInactiveProject(projectName: String): ProjectSettingsDialogPage {
```

Comment:
I don't think we need to assert on the URL when checking a project doesn't exist.